---a/net/minecraft/item/ItemGlassBottle.java
+++b/net/minecraft/item/ItemGlassBottle.java
@@ -9,51 +9,41 @@
 import net.minecraft.util.MovingObjectPosition;
 import net.minecraft.world.World;
 
-public class ItemGlassBottle extends Item
-{
+public class ItemGlassBottle extends Item {
+
     private static final String __OBFID = "CL_00001776";
 
-    public ItemGlassBottle()
-    {
+    public ItemGlassBottle() {
         this.setCreativeTab(CreativeTabs.tabBrewing);
     }
 
-    public ItemStack onItemRightClick(ItemStack itemStackIn, World worldIn, EntityPlayer playerIn)
-    {
+    public ItemStack onItemRightClick(ItemStack itemStackIn, World worldIn, EntityPlayer playerIn) {
         MovingObjectPosition movingobjectposition = this.getMovingObjectPositionFromPlayer(worldIn, playerIn, true);
 
-        if (movingobjectposition == null)
-        {
+        if (movingobjectposition == null) {
             return itemStackIn;
-        }
-        else
-        {
-            if (movingobjectposition.typeOfHit == MovingObjectPosition.MovingObjectType.BLOCK)
-            {
+        } else {
+            if (movingobjectposition.typeOfHit == MovingObjectPosition.MovingObjectType.BLOCK) {
                 BlockPos blockpos = movingobjectposition.getBlockPos();
 
-                if (!worldIn.isBlockModifiable(playerIn, blockpos))
-                {
+                if (!worldIn.isBlockModifiable(playerIn, blockpos)) {
                     return itemStackIn;
                 }
 
-                if (!playerIn.canPlayerEdit(blockpos.offset(movingobjectposition.sideHit), movingobjectposition.sideHit, itemStackIn))
-                {
+                if (!playerIn.canPlayerEdit(blockpos.offset(movingobjectposition.sideHit), movingobjectposition.sideHit, itemStackIn)) {
                     return itemStackIn;
                 }
 
-                if (worldIn.getBlockState(blockpos).getBlock().getMaterial() == Material.water)
-                {
+                if (worldIn.getBlockState(blockpos).getBlock().getMaterial() == Material.water) {
                     --itemStackIn.stackSize;
                     playerIn.triggerAchievement(StatList.objectUseStats[Item.getIdFromItem(this)]);
 
-                    if (itemStackIn.stackSize <= 0)
-                    {
+                    if (itemStackIn.stackSize <= 0) {
                         return new ItemStack(Items.potionitem);
                     }
 
-                    if (!playerIn.inventory.addItemStackToInventory(new ItemStack(Items.potionitem)))
-                    {
+                    if (!playerIn.inventory.addItemStackToInventory(new ItemStack(Items.potionitem))) {
                         playerIn.dropPlayerItemWithRandomChoice(new ItemStack(Items.potionitem, 1, 0), false);
                     }
                 }
@@ -62,4 +62,4 @@
             return itemStackIn;
         }
     }
-}
+}
